src/java/org/apache/cassandra/db/marshal/AbstractType.java

```diff
@@ -529,4 +529,9 @@ public void checkComparable()
     {
         return testAssignment(receiver.type);
     }
+
+    public ByteBuffer unsafeDecompose(Object object)
```

Comment:
I dunno... that a pretty shoot-self-in-foot thing to add the the marshalling API. What's the reasoning?